Makefile

```diff
@@ -82,9 +82,6 @@ race_condition_binary: build
 race_condition_run: race_condition_binary
 	./prometheus.race $(ARGUMENTS)
 
-run: binary
```

Comment:
Can the tutorials be updated instead? They will need updating for the new
config anyways, right? I would not want to this target around just because
a tutorial uses it. "make run" is not recommended practice anymore anyways,
so we should change it in any case.
On May 25, 2015 12:57, "Brian Brazil" <notifications@github.com> wrote:

> In Makefile
> <https://github.com/prometheus/prometheus/pull/728#discussion_r30974348>:
>
> > @@ -82,9 +82,6 @@ race_condition_binary: build
> >  race_condition_run: race_condition_binary
> >  	./prometheus.race $(ARGUMENTS)
> >
> > -run: binary
>
> The tutorials up on boxever.com reference the run target, can we get this
> back in?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/prometheus/pull/728/files#r30974348>.
>